src/main/java/com/commafeed/backend/feed/FeedParser.java

```diff
@@ -170,8 +170,8 @@ private String getContent(SyndEntry item) {
 		}
 		return StringUtils.trimToNull(content);
 	}
-
-	private String getTitle(SyndEntry item) {
+	// This function have been changed to protected for testing purposes

```

Comment:
Same as above

Reply:
I will get it as a question for the teacher next class...